core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -300,6 +413,7 @@ class Partition(val topicPartition: TopicPartition,
     }
   }
 
+  // Visible for testing
```

Comment:
Do we have an annotation like Guava's `@VisibleForTesting`? I always preferred that to comments